packages/path_provider/path_provider_linux/example/test/widget_test.dart

```diff
@@ -9,23 +9,20 @@ import 'dart:async';
 
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:path_provider_linux/path_provider_linux.dart';
 
 import 'package:pathproviderexample/main.dart';
-import 'package:path_provider_platform_interface/path_provider_platform_interface.dart';
 
 void main() {
   group('Test linux path provider example', () {
     setUpAll(() async {
       await WidgetsFlutterBinding.ensureInitialized();
-      PathProviderPlatform.instance = PathProviderLinux();
     });
 
     testWidgets('Finds tmp directory', (WidgetTester tester) async {
       // Build our app and trigger a frame.
       await tester.runAsync(() async {
         await tester.pumpWidget(MyApp());
-        await Future.delayed(Duration(milliseconds: 10));
+        await Future.delayed(Duration(milliseconds: 20));
```

Comment:
Why was this necessary? Does this take more than 10 milliseconds to run?

Reply:
The tests were being flaky, for some reason they were failing on CI. As there are no logical flaws in the test or the app code as far as I can tell I assumed the asynchronous code in the `initState` of `MyApp` that calls out to the plugin to get the various paths had not yet been run. Increasing the timeout seemed to solve the issue. Is there a better way to make sure all asynchronous events are finished being handled in a test?